client/src/components/website/ExplanationText.tsx

```diff
@@ -400,7 +400,7 @@ function useLobbies(appId: string): LobbyV3[] {
   const [lobbies, setLobbies] = React.useState<LobbyV3[]>([]);
   React.useEffect(() => {
     if (appId) {
-      hathoraSdk.lobbyV3.listActivePublicLobbies().then(({ classes }) => {
+      hathoraSdk.lobbiesV3.listActivePublicLobbies().then(({ classes }) => {
```

Comment:
How come the `{ classes }` code was replaced above, but not here? Will this still work? Can we make them consistent?

Reply:
no this won't work that's my miss

Reply:
Oh I see, this was documentation sample code. Thanks for fixing!